src/odemis/gui/util/img.py

```diff
@@ -1265,6 +1265,111 @@ def spectrum_to_export_data(spectrum, raw, unit, spectrum_range):
         return spec_plot
 
 
+def line_to_export_data(spectrum, raw, unit, spectrum_range):
+    """
+    Creates either raw or WYSIWYG representation for the spectrum line data
+
+    spectrum (model.DataArray): spectrum line data
+    raw (boolean): if True returns raw representation
+    unit (string): wavelength unit
+    spectrum_range (list of float): spectrum range
+
+    returns (model.DataArray)
+    """
+
+    if raw:
+        # attach distance as first column
+        data = numpy.zeros(shape=(spectrum.shape[0], spectrum.shape[1]))
+        data = spectrum[:, :, 0]
+        line_length = data.shape[0] * spectrum.metadata[model.MD_PIXEL_SIZE][1]
+        distance_lin = numpy.linspace(0, line_length, data.shape[0])
+        qz_masked = numpy.append(distance_lin.reshape(data.shape[0], 1), data, axis=1)
+        # attach wavelength as first row
+        wavelength_lin = numpy.linspace(spectrum_range[0], spectrum_range[-1], data.shape[1])
+        wavelength_lin = numpy.append([[0]], wavelength_lin.reshape(1, wavelength_lin.shape[0]), axis=1)
```

Comment:
The .reshape() can be simplified and moved to a separate line:
wavelength_lin.shape = (1, wavelength_lin.shape[0])